frontend/src/components/CreateAccountForm.vue

```diff
@@ -7,18 +7,22 @@ import { Form, FormField, type FormSubmitEvent } from '@primevue/forms';
 import { InputText, Message, Button } from 'primevue';
 import { useToast } from 'primevue/usetoast';
 import Toast from 'primevue/toast';
+import {hashPassword} from "@/components/password";
```

Comment:
Missing spaces between brackets and function name

Reply:
Done